js/src/spbc-settings_tab--security_log.js

```diff
@@ -154,6 +154,58 @@ function spbcSecLogsAllowIpCallback(result, data) {
     }
 }
 
+/**
+ * Change role
+ * @param {*} userId
+ * @param {*} role
+ * @param {*} userLogin
+ * @return {boolean}
+ */
+function spbcSecLogsChangeRole(userId, role, userLogin) { // eslint-disable-line no-unused-vars
```

Comment:
Can we make a jest/pupeteer test for this?

Reply:
Yes, will make it in next iteration.